datafusion/core/benches/sql_planner.rs

```diff
@@ -236,40 +144,83 @@ fn criterion_benchmark(c: &mut Criterion) {
         })
     });
 
+    // --- TPC-H ---
+
+    let tpch_ctx = register_defs(SessionContext::new(), tpch_schemas());
```

Comment:
Rather than using the same context for all tests, I made separate contexts for TPCH and TPC-DS